libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -593,6 +621,14 @@ bool AudioIO::StartPortAudioStream(const AudioIOStartStreamOptions &options,
       if( captureDeviceInfo == NULL )
          return false;
 
+      const PaHostApiInfo* hostInfo = Pa_GetHostApiInfo(captureDeviceInfo->hostApi);
+      bool isWASAPI = (hostInfo && hostInfo->type == paWASAPI);
```

Comment:
The constant `paWASAPI` is defined even when not compiling windows, so you will not need a change here to fix the build.